build/Targets/Vsdconfig.targets

```diff
@@ -6,13 +6,17 @@
           BeforeTargets="AfterCompile"
           Inputs="@(VsdConfigXml);$(IntermediateOutputPath)\$(AssemblyName).dll"
           Outputs="$(OutDir)\$(AssemblyName).vsdconfig"
-          Condition="'$(BuildingProject)' == 'true' AND '$(SkipGenerateVsdconfig)' != 'true'">
-    <Exec Command="&quot;$(DevEnvDir)\..\..\VSSDK\VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml -> '%(RelativeDir)%(FileName)%(Extension)', ' ') &quot;$(IntermediateOutputPath)\$(AssemblyName).dll&quot; &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
+          Condition="'$(BuildingProject)' == 'true' AND '@(VsdConfigXml)' != ''">
+    <PropertyGroup>
+      <_VsdConfigTool>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\$(VisualStudioBuildToolsVersion)\tools\VSSDK\bin\vsdconfigtool.exe</_VsdConfigTool>
```

Comment:
Nevermind, I missed that we were explicitly calling a copy in DevEnvDir before.... It would still be useful if they provided a property to easily access the tool though.